lib/Backend/LowerMDShared.cpp

```diff
@@ -9239,8 +9240,11 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
                 instr->Remove();
             }
             break;
-        }
+        }  
 
+    case Js::OpCode::MinNaN:
+    case Js::OpCode::MaxNaN:
+        propagateNaN = true;
```

Comment:
from what i can tell, this is same semantics as for JS, so i think we can use InlineMathMin/InlineMathMax (mostly) as-is